server/views.go

```diff
@@ -53,3 +54,55 @@ func handleAddPlayer(ctx *HandlerCtx) http.HandlerFunc {
 		}
 	}
 }
+
+func handleAddScoreSelectPlayers(ctx *HandlerCtx) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		switch r.Method {
+		case "GET":
+			players := ctx.DB.GetAllPlayers()
+			ctx.Template.Render(w, players)
+		case "POST":
+			r.ParseForm()
+			addScoreURL, err := url.Parse("/score/add/")
+			if err != nil {
+				panic(err)
+			}
+			q := addScoreURL.Query()
+			for name := range r.Form {
+				q.Add("player", name)
+			}
+			addScoreURL.RawQuery = q.Encode()
+			http.Redirect(w, r, addScoreURL.String(), 302)
+		}
+	}
+}
+
+func handleAddScore(ctx *HandlerCtx) http.HandlerFunc {
+	categories := []string{
+		"Military Conflicts",
+		"Wonders",
+		"Treasury Contents",
+		"Civilian Structures",
+		"Scientific Structures",
+		"Commercial Structures",
+		"Guilds",
+	}
+	return func(w http.ResponseWriter, r *http.Request) {
+		switch r.Method {
+		case "GET":
+			players := r.URL.Query()["player"]
+			ctx.Template.Render(w, struct {
+				Players    []string
+				Categories []string
+			}{players, categories})
+		case "POST":
+			r.ParseForm()
+			scores := GetScoresFromForm(r.Form)
+			for k, v := range scores {
+				fmt.Printf("%s %v", k, v)
+			}
+			ctx.DB.SaveScores(scores)
+			http.Redirect(w, r, "/", 302)
```

Comment:
Consider using `http.StatusFound` instead of `302`

Reply:
done